app/models/audio_file_template.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+
+class AudioFileTemplate < BaseModel
```

Comment:
yeah, good point, also looks like some other models (AudioFile has a policy class, but not associated to the model...) are lacking a policy - let me do an audit and clean that up real quick.